web/src/js/components/Dashboard/VideoEngagementComponent.js

```diff
@@ -79,30 +78,104 @@ const VideoEngagementComponent = ({
       closeModal()
     }
   }, [status])
-
   return (
     <div>
-      <ButtonBase onClick={openAd} disabled={!adAvailable}>
-        <MovieFilterIcon
-          {...iconProps}
+      <div
+        ref={poppoverRef}
+        onClick={
+          !adAvailable || !videoAdEligible
+            ? () => setIsPoppoverOpen(true)
+            : undefined
+        }
+        style={{ cursor: 'pointer' }}
+      >
+        <ButtonBase
+          onClick={openAd}
+          disabled={!adAvailable || !videoAdEligible}
+          onMouseEnter={
+            adAvailable && videoAdEligible
+              ? () => setIsPoppoverOpen(true)
+              : undefined
+          }
+          onMouseLeave={() => setIsPoppoverOpen(false)}
+        >
+          <Badge
+            variant="dot"
+            color="primary"
+            classes={{ badge: classes.badgeRoot }}
+            invisible={!adAvailable || !videoAdEligible}
+          >
+            <OndemandVideoIcon
+              style={{ opacity: adAvailable && videoAdEligible ? 1 : 0.54 }}
+              {...iconProps}
+            />
+          </Badge>
+        </ButtonBase>
+      </div>
+      <DashboardPopover
+        style={{
+          pointerEvents: adAvailable ? 'none' : undefined,
```

Comment:
@amaliwanag2 It's possible this is the problem. Should it also respond to `videoAdEligible`? Might want to calculate a prop of `videoAdEnabled` that combines the two for use in the components here.